v2/writer.go

```diff
@@ -3,12 +3,12 @@ package car
 import (
 	"bytes"
 	"context"
+	"github.com/ipld/go-car/v2/internal/index"
```

Comment:
hmmm :) maybe run goimports or gofumpt on all the files to make sure you're not mixing things into the std group?